core/ops_builtin.rs

```diff
@@ -278,6 +279,46 @@ pub fn op_wasm_streaming_set_url(
   Ok(())
 }
 
+#[op2(async)]
+async fn op_wasm_streaming_stream_feed(
+  state: Rc<RefCell<OpState>>,
+  #[smi] rid: ResourceId,
+  #[smi] stream_rid: ResourceId,
+  auto_close: bool,
+) -> Result<(), JsErrorBox> {
+  let wasm_streaming = state
+    .borrow_mut()
+    .resource_table
+    .get::<WasmStreamingResource>(rid)
+    .map_err(|_| JsErrorBox::type_error("stream not found"))?;
+
+  loop {
+    let resource = state
+      .borrow()
+      .resource_table
+      .get_any(stream_rid)
+      .map_err(|_| JsErrorBox::type_error("stream not found"))?;
+    let view = deno_core::BufMutView::new(65536);
```

Comment:
its a shame that read_byob takes this by value instead of by reference, since it might end up being reallocated every loop